packages/main/test/specs/Toolbar.spec.js

```diff
@@ -73,5 +73,15 @@ describe("Toolbar general interaction", () => {
 		await input.setAttribute("value", "0");
 	});
 
+	it("Should not return null upon calling getDomRef for all direct child items", async () => {
```

Comment:
If possible, let's write this new test in cypress

Reply:
Done, no problem at all!